code/modules/projectiles/guns/energy/laser/cog.dm

```diff
@@ -4,7 +4,7 @@
 	icon_state = "cog"
 	item_state = null	//so the human update icon uses the icon_state instead.
 	item_charge_meter = TRUE
-	desc = "A Greyson Positronic design, cheap and widely produced. In the distant past - this was the main weapon of low-rank police forces, billions of copies of this gun were made. They are ubiquitous."
```

Comment:
i actually kind of like this GP flavour description, i think we should keep it

Reply:
Greyson is being axed and re-lored.